frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -159,32 +150,54 @@
 (defn toggle-ignore-click [db _]
   (update-in db [:map :controls :ignore-click] not))
 
-(defn got-feature-info [db [_ request-id priority point info-format response]]
+(defn display-feature-info [db point]
+  (let [responses      (get-in db [:feature-query :responses])
+        response-infos (map
+                        (fn [response]
+                          (if response
+                            (let [{:keys [response info-format]} response]
+                              (case info-format
+                                "text/html" (feature-info-html response)
+                                "application/json" (feature-info-json response)
+                                (feature-info-none)))
+                            {:status :feature-info/empty}))
+                        responses)
+        response-info  (reduce
+                        (fn [acc response-info]
+                          (cond-> acc
+                            (:info response-info)
+                            (update :info str (:info response-info))
+                            (:status response-info)
+                            (assoc :status (:status response-info))
+                            (or (:info response-info) (:info acc))
+                            (dissoc :status)))
```

Comment:
Case in point: what's this doing, exactly?  From what I can see, you're updating a single dict with :info/:status, over a collection (which seems like it would be order-dependent, and perhaps that's the point).  It's probably correct, but it's hard to follow and I'd say at least needs explanation (starting with a name as suggested above, and I'd also suggest comments for each clause)